src/Orchard.Web/Core/Common/Views/Body.Editor.cshtml

```diff
@@ -1,10 +1,16 @@
 ﻿@using Orchard.Utility.Extensions;
 @{
     string editorFlavor = Model.EditorFlavor;
+    var htmlAttributes = (Dictionary<string, object>)Model.HtmlAttributes;
```

Comment:
There should be a "behavior" attached to this HtmlAttributes property that will initialize the dictionary automatically to prevent such tests. Can you try?